applications/UMC-Tokenscribe.md

```diff
@@ -0,0 +1,159 @@
+# Open Grant Proposal
+
+* **Project Name:** UMC - Tokenscribe
+* **Team Name:** UMC Labs
+* **Payment Address:** 1KoTuo8c2YomLXPqLdrPVEiufSDyCwrNGC
+
+## Project Overview :page_facing_up: 
+
+### Overview
+
+#### Introduction
+
+Subscription is a common payment method existed for a long time since credit cards. In the blockchain age, DeFi is the edge of web3 tech to change the world. In the meanwhile, there is no secured subscription feature for users to subscribe to their memberships nor their virtual staffs on-chain.
+As we know, the draft [EIP1337](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1337.md)  proposed a way to implement token subscription for [ERC20](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md)  tokens. But from the view of users, EIP1337 is not the secure way to implement token subscription.
+This application is aimed to propose a specification of secured token subscription which is the key feature of UMC - Universal Membership Community.
+
+#### Integration
+
+The secured token subscription will be implemented with Ink!.
+
+#### Team Interest
+
+We are dapp developers and we want to design a subscription feature for our project to manage users' membership with the secured token subscription. Our project is aimed to build a universal membership network which is can be used for both on-chain and off-chain services, such as Shopping Mall Memberships.
+
+### Project Details 
+
+With EIP1337, all the subscription status-related information is managed by a subscription contract, which will expose the users to the risks just like the situation of subscription in the telecom age and the early age of the internet. Most scamming cases are let users subscribe to some uncertain subscriptions while users cannot cancel those subscriptions due easily.
+The main idea is to track and manage the subscription in token contract rather than subscription contracts to achieve the goal of securing users' assets. 
+
```

Comment:
Where is this going to run? From your interface spec, it looks like just a smart contract.  Are you planning on deploying to a specific parachain?

Reply:
@lamafab  It's spec for developers who want to implement token subscription. Any token contract align with this spec should be run on any parachains.  In this application, we are aiming to propose an interface spec, and a sample contract that follows this interface specification will be delivered as deliverables.

Reply:
It's spec for developers who want to implement token subscription. Any token contract align with this spec should be run on any parachains. In this application, we are aiming to propose an interface spec, and a sample contract that follows this interface specification will be delivered as deliverables.

Reply:
We are heading in different directions compared to #171. In our opinion, everyone should know exactly what he has subscribed to and can unsubscribe from the subscription easily at any time without trying to explore each service from different websites or dapps. As we know, in the real world, lots of subscriptions charge people's money without an easy way to unsubscribe. It would be much more serious in the crypto world. And that's why we want to try in different directions to secure people's crypto assets. In this application, we will demo a contract that follows the spec we proposed.